src/Integration/Transition/MuteWindowDialog.xaml

```diff
@@ -87,7 +89,7 @@
             </Grid>
         </Border>
 
-        <Border Style="{StaticResource BottomBorderStyle}">
+        <Border Name="BorderFalsePositive" Style="{StaticResource BottomBorderStyle}">
```

Comment:
Is this name needed?

Reply:
Yes, to be able to recognize which border was clicked in the code behind. The alternative would be creating different event for each border.

Reply:
This was really confusing because the diff only had one Name change

Reply:
Other borders had already names setup due to being used for visibility.